### PR TITLE
org-pymn part deux

### PR DESCRIPTION
**Pull request by camhashemi**
@leogrim includes some refactoring, including using Set instead of Seq in some places. really happy with the new solution, it only adds a few lines to the shared logic.

going to test the route manually before adding to admin

### CODE REVIEW

**Comment by andrewconner** [comment]
I liked oymn.

**Comment by camhashemi** [comment]
@andrewconner oymn recommends orgs for a user, org-pymn recommends users for an org. oymn is on the way.

File: kifi-backend/modules/abook/app/com/keepit/abook/commanders/AbookOrganizationRecommendationCommander.scala [hunk 1]
```diff
@@ -144,12 +145,19 @@ class AbookOrganizationRecommendationCommander @Inject() (
...
       else {
         val rejectedEmailInviteRecommendations = db.readOnlyReplica { implicit session => organizationEmailInviteRecommendationRepo.getIrrelevantRecommendations(orgId) }
-        val viewersContacts = if (!disclosePrivateEmails) { db.readOnlyMaster { implicit session => contactRepo.getByUserId(viewerId) } } else { Seq.empty }
+        val allContacts = db.readOnlyMaster { implicit session =>
+          if (!disclosePrivateEmails) {
+            contactRepo.getByUserId(viewerId)
+          } else {
+            val relevantEmailAccountIds = relatedEmailAccounts.map { case (id, score) => EmailAccount.fromEmailAccountInfoId(id) }.toSet
+            contactRepo.getByEmailAccountIds(relevantEmailAccountIds)
+          }
```
Comment (leogrim) [inline kifi-backend/modules/abook/app/com/keepit/abook/commanders/AbookOrganizationRecommendationCommander.scala:139]: :+1:

**Comment by leogrim** [comment]
I think you forgot to use `offset` and `limit` in `getRecommendationsForAdmin`, perhaps you can merge these two methods, decide whether to check experiments based on `disclosePrivateEmails` and just pass a different boolean in the controller?

**Comment by camhashemi** [comment]
It should be there? `AbookOrganizationRecommendationCommander[73]`.

But regardless, that's great. Update on the way.

**Pull request merged**